### PR TITLE
13.0 mig purchase tier validation

### PR DESCRIPTION
**Pull request by pegonzalezspesol**
Module is Installable on v13. Work done:
- adaptation of search view
- change of version
- run pre-commit

Currently testing functionality together with base_tier_validation & base_tier_validation_formula

### CODE REVIEW

**Comment by pegonzalezspesol** [comment]
functionality tested following next demo:
https://www.youtube.com/watch?v=ID4JG4jx4Uc

@angelmoya, @lreficent  please review

**Review by HviorForgeFlow — CHANGES_REQUESTED** [review]
add server-ux to oca_dependencies.txt

**Comment by pegonzalezspesol** [comment]
tests added to fulfill code coverage

@angelmoya @lreficent @hveficent please review

**Review by angelmoya — APPROVED** [review]
LGTM :+1:

File: oca_dependencies.txt [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+#
+# To depend on the standard version of sale-workflow, use:
+# sale-workflow
+#
+# To explicitely give the URL of a fork, and still use the version specified in
+# .travis.yml, use:
+# sale-workflow https://github.com/OCA/sale-workflow
+#
+# To provide both the URL and a branch, use:
+server-ux https://github.com/OCA/server-ux 13.0
```
Comment (LoisRForgeFlow) [inline oca_dependencies.txt]: just server-ux is enough, you can remove all the comments I think.

**Review by LoisRForgeFlow — COMMENTED** [review]
Tested and code review, LGTM. Just one comment:

File: oca_dependencies.txt (hunk 1 above)
Comment (HviorForgeFlow) [reply oca_dependencies.txt]: also the url

Comment (pegonzalezspesol) [reply oca_dependencies.txt]: @hveficent done

**Review by HviorForgeFlow — APPROVED** [review]
Functional review :+1:

File: oca_dependencies.txt (hunk 1 above)
Comment (LoisRForgeFlow) [reply oca_dependencies.txt]: @hveficent is it different in v13? until now it it's automatic for OCA repos: https://github.com/OCA/purchase-workflow/blob/12.0/oca_dependencies.txt

Comment (HviorForgeFlow) [reply oca_dependencies.txt]: no, like you said, simple server-ux, for example.

Comment (lreficent) [reply oca_dependencies.txt]: @pegonzalezspesol sorry for bothering you with this minor thing, but I think is better to be align in all repos, so can you update the file again to be just "sever-ux"?

**Review by LoisRForgeFlow — APPROVED** [review]
ready to go!

**Pull request merged**